src/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useCallback } from 'react';
```

Comment:
You may want to check out src/Icon/Icon.tsx

Reply:
> You may want to check out src/Icon/Icon.tsx

Regarding specifying template types for `createComponent`? Or was there something else I'm missing?

Reply:
Nothing in particular, just FYI for you so you can reference it.